variables.tf

```
@@ -51,6 +51,12 @@ variable "availability_zone" {
   description = "Optional parameter to place Amazon Redshift cluster instances in a specific availability zone. If left empty, will place randomly"
 }
 
+variable "availability_zone_relocation_enabled" {
+  type        = bool
+  default     = false
+  description = "If true, the cluster can be relocated to another availabity zone, either automatically by AWS or when requested"
```

Let's expand on this input

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/redshift_cluster#availability_zone_relocation_enabled

```suggestion
  description = "Whether or not the cluster can be relocated to another availability zone, either automatically by AWS or when requested. Available for use on clusters from the RA3 instance family."
```